tests/CMakeLists.txt

```diff
@@ -19,7 +20,6 @@ target_link_libraries(models_test
   ${COMPILER_SUPPORT_LIBRARIES}
   ${ARMADILLO_LIBRARIES}
   ${Boost_FILESYSTEM_LIBRARY}
-  ${Boost_UNIT_TEST_FRAMEWORK_LIBRARY}
   ${Boost_SYSTEM_LIBRARY}
   ${Boost_SERIALIZATION_LIBRARY}
```

Comment:
```suggestion
```
Here's another thing we could remove.  I think in these suggestions I haven't caught everything, so maybe there is more to do too.